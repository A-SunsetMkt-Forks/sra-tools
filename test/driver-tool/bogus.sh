#!/bin/bash

bin_dir=$1

echo "testing expected output for unknown tool"

TEMPDIR=.

mkdir -p actual

output=$(NCBI_SETTINGS=${TEMPDIR}/tmp.mkfg \
	PATH="${bin_dir}:$PATH" \
	SRATOOLS_IMPERSONATE=rcexplain \
	${bin_dir}/sratools 2>actual/bogus.stderr ; \
	diff expected/bogus.stderr actual/bogus.stderr)

res=$?
if [ "$res" != "0" ];
	then echo "Driver tool test unknown tool, res=$res output=$output" && exit 1;
fi

echo Driver tool test unknown tool is finished