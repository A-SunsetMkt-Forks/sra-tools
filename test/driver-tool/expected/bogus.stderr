An error occured: unrecognized tool rcexplain
If this continues to happen, please contact the SRA Toolkit at https://trace.ncbi.nlm.nih.gov/Traces/sra/
